zebrad/tests/acceptance.rs

```diff
@@ -266,7 +268,17 @@ fn version_args() -> Result<()> {
 }
 
 #[test]
-fn valid_generated_config() -> Result<()> {
+fn valid_generated_config_test() -> Result<()> {
+    // Unlike the other tests, these tests can not be run in parallel, because
+    // they use the generated config. So parallel execution can cause port and
+    // cache conflicts.
```

Comment:
Nice one, i missed that.

Reply:
It was already implemented by the test function, but it's good to document it in a comment.